app/components/multi-select-dropdown/multi-select-dropdown.component.html

```diff
@@ -29,7 +29,7 @@
                         'multiselect-dropdown__option--selected':
                             vm.isOptionSelected(option)}"
                     ng-click="vm.onOptionSelection(option)">
-                    <i class="fas fa-check" ng-if="vm.isOptionSelected(option)"></i>
+                    <i class="fa fas fa-check" ng-if="vm.isOptionSelected(option)"></i><span ng-if="vm.isOptionSelected(option)">&nbsp;</span>
```

Comment:
Can we use padding-left on the label instead of an empty space character?

Reply:
That was my original solution, but I wasn't able to get it to actually apply...added gap instead & that's working. ⭐